pkg/tfgen/parse/meta.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2016-2024, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package parse
+
+import (
+	"github.com/yuin/goldmark"
+	meta "github.com/yuin/goldmark-meta"
```

Comment:
nice find! Feel free to ignore me on this, but is there a reason we need to alias this here? I have to admit I went back and forth a bit on "meta" being a bit overloaded as a word in general, and leaving it as `goldmark-meta` at least tells the future maintainer that it's a parser thing.

Reply:
We don't need the alias... it's actually implied. `-` is not valid in Go's identifier syntax, so it just drops the `goldmark-` part:

```go
import "github.com/yuin/goldmark-meta"

...

func myFunc(meta.Extension) {}
```